compiler/test/dotty/tools/dotc/reporting/ErrorMessagesTests.scala

```diff
@@ -1527,4 +1527,23 @@ class ErrorMessagesTests extends ErrorMessagesTest {
       val NotAnExtractor(tree) = messages.head
       assertEquals("Foo", tree.show)
     }
+
+  @Test def memberWithSameNameAsStatic() =
+    checkMessagesAfter(CheckStatic.name) {
+      """
+        |import scala.annotation.static
+        |class Camp {
+        |  val name = ""
+        |}
+        |object Camp {
+        |  @static val name = ""
+        |}
+      """.stripMargin
+    }.expect { (ictx, messages) =>
+      implicit val ctx: Context = ictx
```

Comment:
I believe the implicit `ctx` is not needed for your test

Reply:
Indeed it wasn't needed, thanks!